docker-compose.yml

```diff
@@ -15,17 +15,17 @@ services:
     # volumes:
     #   - /app/node_modules
     #   - ./mock-api:/app
-  web_test:
-    build:
-      context: .
-      dockerfile: Dockerfile.dev
-    volumes:
-      - /app/node_modules
-      - .:/app
-    # environment:
-    #   - CI=true
-    command: yarn test
-    restart: on-failure
+  # web_test:
```

Comment:
@luixlive is the intent to no longer run the test suite in a container?

Reply:
Sorry, I'll uncomment those. It really makes my docker instance slower. Every save I do to a file reruns a lot of tests, taking over all the resources my docker has instead of leaving those to the app itself.